src/Diagnostics.RuntimeHost/appsettings.json

```diff
@@ -149,6 +149,9 @@
   },
   "AutoHealMonitoringSettings": {
     "IsEnabled": true,
-    "IterationDelayInSeconds" : 1800
+    "IterationDelayInSeconds": 1800
+  },
+  "DiagnosticTranslator": {
```

Comment:
I think it would be helpful to add a On/Off flag for Translation. Two Benefits:
- In case things dont go as planned, we can quickly turn this feature off
- For other S-Clouds, we dont want to regress the APIs and we can keep this off to begin with there.

Reply:
Very good point!!

Reply:
Please also add these settings to Dev KeyVault to maintain consistency :) Thanks

Reply:
Will do @rekhaswaminathan:)